Lib/fontTools/cu2qu/cu2qu.py

```diff
@@ -77,6 +77,9 @@ def calc_cubic_parameters(p0, p1, p2, p3):
 @cython.cfunc
 @cython.locals(p0=cython.complex, p1=cython.complex, p2=cython.complex, p3=cython.complex)
 def split_cubic_into_n_iter(p0, p1, p2, p3, n):
+    """Given four vectors representing control points of a cubic curve,
```

Comment:
are bezier control "points" really "vectors"? I'd call them just "points" (aka locations in bidimensional space relative to origin). "vector" implies a direction and magnitude, a displacement, a subtraction between two points, an arrow..
At least in computational geometry.. Or do you mean "vector" here as an "array" of coordinates?

Reply:
+1 for "points".

Reply:
Yeah, I meant they're not scalars. I think I would prefer to call them "complex numbers" because that clarifies input expectations and we already say that they're "points" later in the sentence.

Reply:
if we go with #1947, then in the first line (the compact description) you'd call them points and in the Args section you'd annotate them with their complex type.